src/My.Extensions.Localization.Json/JsonMvcDataAnnotationsMvcBuilderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Microsoft.Extensions.Localization;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class JsonLocalizationMvcDataAnnotationsExtensions
+    {
+        public static IMvcBuilder AddDataAnnotationsJsonLocalization(this IMvcBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+            builder.AddDataAnnotationsLocalization(o =>
```

Comment:
Use `options` instead of `o` for clarity and I'm fine to approve all the changes except the MVC package, but it will be ready for merge here or somewhere else

thanks